docs/data-sources/access_interface_override.md

```diff
@@ -0,0 +1,76 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Access Policies"
+layout: "aci"
+page_title: "ACI: aci_access_interface_override"
+sidebar_current: "docs-aci-data-source-aci_access_interface_override"
+description: |-
+  Data source for ACI Access Interface Override
+---
+
+# aci_access_interface_override #
+
+Data source for ACI Access Interface Override
+
+## API Information ##
+
+* Class: [infraHPathS](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/infraHPathS/overview)
+
+* Supported in ACI versions: 1.1(1j) and later.
+
+* Distinguished Name Format: `uni/infra/hpaths-{name}`
+
+## GUI Information ##
+
+* Locations:
+  - `Fabric -> Access Policies -> Interfaces -> Leaf Interfaces -> Overrides`
+  - `Fabric -> Access Policies -> Interfaces -> Spine Interfaces -> Overrides`
+
+## Example Usage ##
+
+```hcl
+
+data "aci_access_interface_override" "example" {
+  name = "host_path_selector"
+}
+
+```
+
+## Schema ##
+
+### Required ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - The distinguished name (DN) of classes below can be used but currently there is no available resource for it:
+    - [infraInfra](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/infraInfra/overview)
+
+  - Default: `uni/infra`
+  
+* `name` (name) - (string) The name of the Access Interface Override object.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the Access Interface Override object.
+* `annotation` (annotation) - (string) The annotation of the Access Interface Override object.
+* `description` (descr) - (string) The description of the Access Interface Override object.
+* `name_alias` (nameAlias) - (string) The name alias of the Access Interface Override object.
+* `owner_key` (ownerKey) - (string) The key for enabling clients to own their data for entity correlation.
+* `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
+
+* `relation_to_host_paths` - (list) A list of Relation To Host Paths (ACI object [infraRsHPathAtt](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/infraRsHPathAtt/overview)) pointing to  (ACI Object [fabricPathEp](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fabricPathEp/overview)).
```

Comment:
We are going to keep it like that for the moment. The rendering will probably remove this extra white space. The problem is we need to add a bunch of conditions in template to remove just that one space. It happens because the ACI Object does not have a resource yet in the new plugin framework.